### PR TITLE
Add the missing relations to the DCAs

### PR DESCRIPTION
**Pull request by aschempp**
https://github.com/contao/contao/pull/7563 for Contao 5.3

### CODE REVIEW

**Comment by ausi** [comment]
> for Contao 5.3

You need to switch the branch to `5.3` then I think.

File: core-bundle/contao/dca/tl_image_size.php [hunk 1]
```diff
@@ -155,7 +155,8 @@
 			'inputType'               => 'checkboxWizard',
 			'options_callback'        => array('tl_image_size', 'getFormats'),
 			'eval'                    => array('multiple'=>true),
-			'sql'                     => "varchar(1024) NOT NULL default ''"
+			'sql'                     => "varchar(1024) NOT NULL default ''",
+			'relation'                => array('table'=>'tl_image_size', 'type'=>'hasMany', 'load'=>'lazy')
```
Comment (leofeyer) [inline core-bundle/contao/dca/tl_image_size.php]: This seems odd. The `getFormats()` method returns an array of strings and not a collection of image size models. Are you sure this is correct?

Comment (aschempp) [reply core-bundle/contao/dca/tl_image_size.php]: you're right, that makes no sense.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Pull request merged**